src/utilities/utility.ts

```diff
@@ -130,6 +130,12 @@ export function annotateChildPods(kubectl: Kubectl, resourceType: string, resour
     return commandExecutionResults;
 }
 
+export function normaliseWorkflowStrLabel(workflowStr: string): string {
```

Comment:
Could replace term `workflowStr` by `workflowName` in method and parameter names?